### DISCUSSION TITLE
请问对于那种不换行的标题ppstructure能做到比较好的识别吗？

### DISCUSSION THREAD

**Discussion by ZzYAmbition** [PaddlePaddle/PaddleOCR#14193] [Q&A]
![image](https://github.com/user-attachments/assets/999af72d-57c6-4ebc-b157-5c6ab7383f6a)
如图所示的这种类型的标题。我跑了一下ppstructure，是没办法识别出来的。请问有比较好的方法吗？

**Comment by GreatV** [comment]
可以试试其他版面识别算法模型
